### DISCUSSION TITLE
Welcome to recastui Discussions!

### DISCUSSION THREAD

**Discussion by praneybehl** [celerity-one/celerity#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abhigyantrips** [comment]
just wanna say i'm actually stoked for this project - the base design really gives me vibes of the [Carbon Design System](https://carbondesignsystem.com), and i'm looking forward to using it :)
